src/fdb5/database/ArchiveVisitor.h

```diff
@@ -43,6 +43,8 @@ class ArchiveVisitor : public BaseArchiveVisitor {
     const void *data_;
     size_t size_;
 
+    ArchiveCallback callback_;
```

Comment:
Is there value in taking this by reference rather than by copy?

Reply:
Probably ought to be using reference essentially everywhere, updated to do that.